experimental/Schemes/src/ToricBlowups/methods.jl

```diff
@@ -1,33 +1,33 @@
 @doc raw"""
-    strict_transform(f::ToricBlowupMorphism, I::MPolyIdeal) -> MPolyIdeal
-    strict_transform(f::ToricBlowupMorphism, g::MPolyDecRingElem) -> MPolyDecRingElem
+    strict_transform(phi::ToricBlowupMorphism, I::MPolyIdeal) -> MPolyIdeal
+    strict_transform(phi::ToricBlowupMorphism, f::MPolyDecRingElem) -> MPolyDecRingElem
 
-Let $f\colon Y \to X$ be the toric blowup corresponding to a star
-subdivision along a ray. Let $R$ and $S$ be the Cox rings of $X$ and
-$Y$, respectively.
+Let $\varphi\colon Y \to X$ be the toric blowup corresponding to a star
+subdivision along a primitive vector in the support of the fan of $X$.
+Let $R$ and $S$ be the Cox rings of $X$ and $Y$, respectively.
 Here "strict transform" means the "scheme-theoretic closure of the
 complement of the exceptional divisor in the scheme-theoretic inverse
 image".
 This function returns a homogeneous ideal (or a polynomial generating a
-principal ideal) in $S$ corresponding to the strict transform under $f$
+principal ideal) in $S$ corresponding to the strict transform under $\varphi$
 of the closed subscheme of $X$ defined by the homogeneous ideal $I$ (or
-by the homogeneous principal ideal generated by $g$) in $R$.
+by the homogeneous principal ideal generated by $f$) in $R$.
 
 This is implemented under the following assumptions:
   * the variety $X$ has no torus factors (meaning the rays span
     $N_{\mathbb{R}}$).
```

Comment:
Maybe the following makes sense too/is cleaner?
```suggestion
This is implemented under the assumption that the variety $X$ has no torus factor (meaning its rays span
$N_{\mathbb{R}}$).
```

Reply:
Changed "assumptions" -> "assumption". The rest I prefer to keep as it is.